lib/srv/db/common/permissions/presets.go

```diff
@@ -0,0 +1,54 @@
+// Teleport
+// Copyright (C) 2024 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package permissions
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	dbobjectimportrulev1pb "github.com/gravitational/teleport/api/gen/proto/go/teleport/dbobjectimportrule/v1"
+	"github.com/gravitational/teleport/api/types/databaseobjectimportrule"
+	"github.com/gravitational/teleport/api/types/label"
+)
+
+// NewPresetImportAllObjectsRule creates new "import_all_objects" database object import rule, which applies `kind: <object kind>` label to all database objects.
```

Comment:
I think this is a good argument for adding templating capabilities into those values, would you agree?

Reply:
> I think this is a good argument for adding templating into label values, would you agree?

ya, that would be cool

Reply:
This should be relatively simple - I have a draft implementation ready - but I think it would be worth raising as a separate PR. This one is pretty big already.

Reply:
@greedy52  https://github.com/gravitational/teleport/pull/38630